spec/integration/compose/rails_debugger_spec.rb

```diff
@@ -2,7 +2,6 @@
 
 RSpec.describe "Compose rails debugger" do
```

Comment:
MapIt is not a Rails app, so this description is incorrect if we include MapIt again. I think the suspicion that this was a "Rails-only test" was the reason for the ignoring.